android/src/com/mapswithme/maps/bookmarks/ChooseBookmarkCategoryFragment.java

```diff
@@ -142,7 +126,12 @@ public void onCategorySet(int categoryPosition)
   @Override
   public void onCategoryCreate()
   {
-    EditTextDialogFragment.show(getString(R.string.bookmark_set_name), null,
-                                getString(R.string.ok), null, this);
+    EditTextDialogFragment dialogFragment = EditTextDialogFragment.show(getString(R.string.bookmark_set_name),
+                                                              null,
```

Comment:
nit: why is it not aligned with getString?

Reply:
What formatting should I use in case of long lines?

Option 1: Continue even if line is longer than 100 chars

![formatting-1](https://user-images.githubusercontent.com/720808/145168948-7e3f1d0c-881b-4bd1-a492-c64e42f281a9.png)

Option 2: Add line break after `=` sign

![formatting-2](https://user-images.githubusercontent.com/720808/145168976-295c7d20-1616-4cd1-bb9e-75d6c88ccc5b.png)

Reply:
Both look better than a previous variant. A bottom one could be easier to read.

Reply:
Okay, use option 2 formatting